tests/integration/models/test_entity_set.py

```diff
@@ -0,0 +1,135 @@
+"""Tests for the entity_set module
+
+Validations that need to be enforced programmatically as they
+  do not raise an exception with sqlalchemy
+  * EntitySet.id should not allow the empty string
+  * EntitySet.entity_ids values cannot exceed 36 characters
+  * EntitySet.entity_ids array should not be empty
```

Comment:
This is probably something I missed during design. Would it ever be possible that upon a DR, a cohort that previously had IDs when running the filters query had none? Or similarly, would t ever be possible that we ended up getting 0 cases qualifying for a filters query as the result of a users auth changes?

If either of these is possible, we probably want to support empty sets.

Reply:
I'm fine with empty sets. It currently supports that so I'll remove that comment.